Sources/NIO/BSDSocketAPI.swift

```diff
@@ -12,11 +12,20 @@
 //
 //===----------------------------------------------------------------------===//
 
-#if os(Windows)
+#if os(iOS) || os(macOS) || os(tvOS) || os(watchOS)
+@_exported
+import Darwin.C
+#elseif os(Windows)
+@_exported
```

Comment:
Should we be doing an exported import here? I think we regret doing it for libc.

Reply:
Yeah, this is a difficult one - maintain parity or do something unfortunate.  I suppose that maintaining parity here is a little less important, particularly since most of the interfaces for networking are not part of the C library anyways.